plugin/db/mysql/restore.go

```diff
@@ -304,6 +296,27 @@ func GetBinlogReplayList(startBinlogInfo api.BinlogInfo, binlogDir string) ([]st
 	return binlogReplayList, nil
 }
 
+func getBinlogMetaSlice(metaList []binlogFileMeta, startSeq, targetSeq int64) ([]binlogFileMeta, error) {
```

Comment:
Need to find a better name. getBinlogMetaSlice doesn't tell anything till someone looks at its implementation details. The name has to be descriptive and state exactly what it does.

Reply:
Changed to `getMetaReplayList`, WDYT?

Reply:
SG

Reply:
SGTM.